### PR TITLE
Goblin traditional robes

### PR DESCRIPTION
**Pull request by ErhardSteinhauer**
## About the PR
Adds 2 outer clothing items and recipes for crafting those: goblin robes and goblin eva robes.

## Why / Balance
Important items for goblin identity

## How to test
1. Spawn goblin robes through entity spam menu
2. Use construction menu to craft them

## Media

image
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
:cl: erhardsteinhauer
- add: Added goblin robes, goblin EVA robes, and recipes to craft them.

### CODE REVIEW

File: Resources/Prototypes/_NF/tags.yml [hunk 1]
```diff
@@ -87,3 +87,6 @@
   
 - type: Tag
   id: SpeedLoaderRifleHeavy
+
+- type: Tag
+  id: ClothingRobesGoblin
```
Comment (whatston3) [inline Resources/Prototypes/_NF/tags.yml:92]: Ominous.

File: Resources/Prototypes/_NF/Entities/Clothing/OuterClothing/goblin_outerclothing.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    slot: head
+  - type: Construction
+    graph: ClothingOuterCoatRobesGoblinGraph
+    node: ClothingOuterCoatRobesGoblinNode
+
+- type: entity
+  parent: ClothingOuterEVASuitBase
+  id: ClothingOuterEVASuitGoblin
+  name: goblin EVA suit
+  description: Traditional robes of goblinkind turned in EVA.
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Entities/Clothing/OuterClothing/goblin_outerclothing.yml:25]: "turned into EVA" is more acceptable, but I think this is better:
```suggestion
  description: Traditional robes of goblinkind made EVA-capable. Very regal.
```

File: Resources/Prototypes/_NF/Recipes/Crafting/goblin_clothes.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  objectType: Item
+
+- type: construction
+  name: goblin traditional EVA robes
+  id: ClothingOuterEVASuitGoblinRecipe
+  graph: ClothingOuterEVASuitGoblinGraph
+  startNode: start
+  targetNode: ClothingOuterEVASuitGoblinNode
+  category: construction-category-clothing
+  description: Traditional robes of goblinkind turned in EVA. Very regal.
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Recipes/Crafting/goblin_clothes.yml:22]:
```suggestion
  description: Traditional robes of goblinkind made EVA-capable. Very regal.
```

File: Resources/Prototypes/_NF/Entities/Clothing/OuterClothing/goblin_outerclothing.yml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  - type: ToggleableClothing
+    clothingPrototype: ClothingHeadEVAHelmetGoblin
+    slot: head
+  - type: ContainerContainer
+    containers:
+      toggleable-clothing: !type:ContainerSlot {}
+  - type: StaticPrice
+    price: 65
+  - type: Construction
+    graph: ClothingOuterEVASuitGoblinRecipe
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Entities/Clothing/OuterClothing/goblin_outerclothing.yml:44]: Think this is correct off the top of my head, matches the non-EVA version.
```suggestion
    graph: ClothingOuterEVASuitGoblinGraph
```

**Review by whatston3 — CHANGES_REQUESTED** [review]
Looks reasonable, left a few comments, they should fix your linter error.

I think "traditional goblin hood/robe" and "traditional goblin EVA hood/robe" read a bit better, really nitpicky, but I'll leave it up to you.

**Comment by whatston3** [comment]
Works well with suggested changes, will approve when they're in.

File: Resources/Prototypes/_NF/tags.yml (hunk 1 above)
Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/tags.yml:92]: Even goblin clothes are capable of robbing.

**Review by whatston3 — APPROVED** [review]
Thanks, Erhard.

**Pull request merged**